maintenance/wikia/lyrics/storage/schema.xml

```diff
@@ -1,130 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-<schema name="lyrics" version="1.5">
```

Comment:
Is this schema not used, or just somewhere else?

Reply:
Solr schema is kept in Chef